src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -7681,7 +7681,7 @@ paths:
             trace.
       security:
       - jwt: []
-      summary: 'List all or cluster specific NFS-Ganesha exports '
+      summary: List all or cluster specific NFS-Ganesha exports
```

Comment:
this should actually come from here: https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/controllers/nfs.py#L117

and then update the spec using `tox -e openapi-fix`

https://docs.ceph.com/en/latest/dev/developer_guide/dash-devel/#rest-api-documentation

Reply:
@nizamial09 hi Nizamudeen, thanks for the pointers. in order to get `tox -e openapi-fix` to run on my arch box, i had to regenerate `gateway_pb2.py` with the protobuf on my system, and relax `requirements.txt` to stop pinning grpcio and grpcio-tools, otherwise they failed to compile with GCC-15.

but the generated `openapi.yaml` is quite different from the original one. to avoid the code churn, i manually edited this file.

Reply:
thanks @nizamial09 , ahh, so it's #62239 if i am not mistaken. looking forward to having it in `main` soon!

Reply:
oh yep, that's the one.